lib/smack/SmackRep.cpp

```diff
@@ -361,31 +546,66 @@ string SmackRep::getString(const llvm::Value* v) {
 const Expr* SmackRep::cast(const llvm::Instruction* I) {
   return cast(I->getOpcode(), I->getOperand(0), I->getType());
 }
-
 const Expr* SmackRep::cast(const llvm::ConstantExpr* CE) {
   return cast(CE->getOpcode(), CE->getOperand(0), CE->getType());
 }
 
+string SmackRep::bopName(unsigned operand1, unsigned operand2, const string& operation)
+{
+  stringstream s;
+  s << operation << ".i" << operand1 << "i" << operand2;
+  return s.str();
+}
+
+string SmackRep::bopName(unsigned operand1, const string& operation)
+{
+  stringstream s;
+  s << operation << ".i" << TRUNC(operand1);
+  return s.str();
+}
+
+string SmackRep::uopName(unsigned operand, const string& operation, unsigned debug)
```

Comment:
This method is redundant with `int_type` and trivial concatenation, and is only used once. Just get rid of it.